docs/reference-guides/cluster-configuration/rancher-server-configuration/rke2-cluster-configuration.md

```diff
@@ -15,81 +15,81 @@ You can configure the Kubernetes options in one of the two following ways:
 - [Rancher UI](#configuration-options-in-the-rancher-ui): Use the Rancher UI to select options that are commonly customized when setting up a Kubernetes cluster.
 - [Cluster Config File](#cluster-config-file-reference): Instead of using the Rancher UI to choose Kubernetes options for the cluster, advanced users can create an RKE2 config file. Using a config file allows you to set many additional [options](https://docs.rke2.io/install/configuration) available for an RKE2 installation.
 
-## Editing Clusters with a Form in the Rancher UI
+## Editing Clusters in the Rancher UI
+
+Rancher UI provides two ways to edit cluster: editing clusters with a form or in YAML.
+
+### Editing Clusters with a Form
+
+The form contains the most common and necessary options for clusters.
 
 To edit your cluster,
 
 1. In the upper left corner, click **☰ > Cluster Management**.
 1. Go to the cluster you want to configure and click **⋮ > Edit Config**.
 
-## Editing Clusters with YAML
+### Editing Clusters in YAML
```

Comment:
I'm pausing here but changes suggested in the other file should also be performed here whenever applicable.